R/msaAttribute.R

```diff
@@ -561,7 +561,7 @@ msaAttribute <- function(jaspResults, dataset, options, ...) {
       AAA[["AllVsStandard"]] <- tableAllVsStandard
     }
 
-    if (options$standard != "" & options$PositiveRef != "")
+    if (options$standard != "" & options$PositiveRef != "" & length(measurements) > 1)
```

Comment:
```suggestion
    if (options$standard != "" && options$PositiveRef != "" && length(measurements) > 1)
```
please make sure to use `&&` and `||` consistently everywhere.